docs/project/set-up-git.md

```diff
@@ -195,7 +195,7 @@ the branch to your fork on GitHub:
 
     ```bash
     $ git branch
-    * dry-run-test
+    \* dry-run-test
```

Comment:
ow, that's weird that it doesn't support that. To prevent having the back-slash showing up in the output, can you remove the "fences", and indent the whole code-block with 4 extra spaces? That way it renders properly, without the back-slash visible;

<img width="715" alt="screen shot 2016-05-20 at 16 01 17" src="https://cloud.githubusercontent.com/assets/1804568/15430126/3d1c3f1c-1ea4-11e6-91a7-8d78192a17ea.png">

Diff is below:

```patch
diff --git a/docs/project/set-up-git.md b/docs/project/set-up-git.md
index e7f13fd..70cb6b1 100644
--- a/docs/project/set-up-git.md
+++ b/docs/project/set-up-git.md
@@ -193,11 +193,9 @@ the branch to your fork on GitHub:
 
 3. Verify you are in your new branch.
 
-    ```bash
-    $ git branch
-    \* dry-run-test
-      master
-    ```
+        $ git branch
+        * dry-run-test
+          master
 
     The current branch has an * (asterisk) marker. So, these results shows you
     are on the right branch.
@@ -265,15 +263,13 @@ the branch to your fork on GitHub:
     Git prompts you for your GitHub username and password. Then, the command
     returns a result.
 
-    ```bash
-    Counting objects: 13, done.
-    Compressing objects: 100% (2/2), done.
-    Writing objects: 100% (3/3), 320 bytes | 0 bytes/s, done.
-    Total 3 (delta 1), reused 0 (delta 0)
-    To https://github.com/moxiegirl/docker.git
-     \* [new branch]      dry-run-test -> dry-run-test
-    Branch dry-run-test set up to track remote branch dry-run-test from origin.
-    ```
+        Counting objects: 13, done.
+        Compressing objects: 100% (2/2), done.
+        Writing objects: 100% (3/3), 320 bytes | 0 bytes/s, done.
+        Total 3 (delta 1), reused 0 (delta 0)
+        To https://github.com/moxiegirl/docker.git
+         * [new branch]      dry-run-test -> dry-run-test
+        Branch dry-run-test set up to track remote branch dry-run-test from origin.
 
 9. Open your browser to GitHub.
 
```